modules/rosa-sts-aws-requirements-attaching-boundary-policy.adoc

```diff
@@ -0,0 +1,371 @@
+While the ROSA (`rosa`) CLI offers a permission boundary function, it applies to all roles and not just the installer role, which means it does not work with the provided permission boundary policies (which are only for the installer role).
+====
+
+.Prerequisites
+
+* You have an AWS account.
+* You have the permissions required to administer AWS roles and policies.
+* You have installed and configured the latest AWS (`aws`) and ROSA (`rosa`) CLIs on your workstation.
+* You have already prepared your ROSA account-wide roles, includes the installer role, and the corresponding policies. If these do not exist in your AWS account, see "Creating the account-wide STS roles and policies" in _Additional resources_.
+
+.Procedure
+
+. Prepare the policy file by entering the following command in the `rosa` CLI:
++
+[source,terminal]
+----
+$ curl -o ./rosa-installer-core.json https://raw.githubusercontent.com/openshift/managed-cluster-config/master/resources/sts/4.16/sts_installer_core_permission_boundary_policy.json
+----
+
+. Create the policy in AWS and gather its Amazon Resource Name (ARN) by entering the following command:
++
+[source,terminal]
+----
+$ aws iam create-policy \
+--policy-name rosa-core-permissions-boundary-policy \
+--policy-document file://./rosa-installer-core.json \
+--description "ROSA installer core permission boundary policy, the minimum permission set, allows BYO-VPC, disallows PrivateLink"
+----
++
+.Example output
+[source,terminal]
+----
+{
+    "Policy": {
+        "PolicyName": "rosa-core-permissions-boundary-policy",
+        "PolicyId": "<Policy ID>",
+        "Arn": "arn:aws:iam::<account ID>:policy/rosa-core-permissions-boundary-policy",
+        "Path": "/",
+        "DefaultVersionId": "v1",
+        "AttachmentCount": 0,
+        "PermissionsBoundaryUsageCount": 0,
+        "IsAttachable": true,
+        "CreateDate": "<CreateDate>",
+        "UpdateDate": "<UpdateDate>"
+    }
+}
+----
+. Add the permission boundary policy to the installer role you want to restrict by entering the following command:
++
+[source,terminal]
+----
+$ aws iam put-role-permissions-boundary \
+--role-name ManagedOpenShift-Installer-Role \
+--permissions-boundary arn:aws:iam::<account ID>:policy/rosa-core-permissions-boundary-policy
+----
+
+. Display the installer role to validate attached policies (including permissions boundary) by entering the following command in the `rosa` CLI:
++
+[source,terminal]
+----
+$ aws iam get-role --role-name ManagedOpenShift-Installer-Role \
+--output text | grep PERMISSIONSBOUNDARY
+----
++
+.Example output
+[source,terminal]
+----
+PERMISSIONSBOUNDARY	arn:aws:iam::<account ID>:policy/rosa-core-permissions-boundary-policy	Policy
+----
++
+For more information, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html[Permissions boundaries for IAM entities] (AWS documentation).
```

Comment:
This link seems a bit hidden with the surrounding content. I still think adding it to the add resources section would make the link more prominent.

Reply:
moved this link to add resources, thanks